ia/test/unitTest.py

```
@@ -0,0 +1,29 @@
+import unittest
+import sys
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent.parent / 'src'))
+
+import message as msg
+
+
+class SimpleEncodingTestCase(unittest.TestCase):
+    def test_encrypt_basic(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_encrypt_basic`

Add return type annotation: `None`

(ANN201)